pkgs/top-level/all-packages.nix

```diff
@@ -2881,6 +2881,8 @@ in
   tsm-client = callPackage ../tools/backup/tsm-client { jdk8 = null; };
   tsm-client-withGui = callPackage ../tools/backup/tsm-client { };
 
+  trac = pythonPackages.callPackage ../tools/misc/trac { };
```

Comment:
would be nice nice for upstream to have a python3 supported installation....
```suggestion
  trac = pythonPackages.callPackage ../tools/misc/trac { };
```

Reply:
Yeah, sadly not supported.